docs/DeveloperGuide.md

```diff
@@ -1,38 +1,286 @@
-# Developer Guide
+<div align="center">
+<h2> MoneyMind Developer Guide</h2>
+<h3> "Mind your Money" </h3>
+</div>
+
+<div align="left">
 
 ## Acknowledgements
```

Comment:
![image](https://user-images.githubusercontent.com/88263677/229004359-8d36d945-da85-451b-bfab-f59f0300253b.png)

The text formatting is not showing up. Might be because there are no newlines.